Makefile

```diff
@@ -0,0 +1,16 @@
+all: server test clean
+
+server:
+	@echo "\nSetting up the server\n"
+	@gunicorn server &
+	@sleep 1
+
+test:
+	@echo "\n\n\nRunning test:\n"
+	@python test.py
+	@sleep 1
```

Comment:
Overwise, the whole iutput is messy: test.py begins to print, followed up by output from gunicorn. Hard to keep track of what's going on.